catroid/src/main/java/org/catrobat/catroid/content/Sprite.java

```diff
@@ -55,9 +57,14 @@
 	private static final long serialVersionUID = 1L;
```

Comment:
I'd suggest to declare the Sprite class as abstract, but I'm not sure if this would be backwards compatible with older Catrobat language versions (XStream?).